src/schema/v2/me/index.ts

```diff
@@ -53,6 +54,22 @@ import { SavedArtworks } from "./savedArtworks"
 import { ShowsByFollowedArtists } from "./showsByFollowedArtists"
 import { WatchedLotConnection } from "./watchedLotConnection"
 
+/**
+ * @deprecated: Please use the CollectorProfile type instead of adding fields to me directly.
+ */
```

Comment:
For some reason, we started adding collector profile attributes to `me` instead of `me.collectorProfile`. I've added this helper function with a deprecation warning to make sure we add these fields to `me.collectorProfile` in the future.